doc/magic_cards_notes.md

```diff
@@ -1104,29 +1165,35 @@ hf 14a raw -s -c -t 1000 CF000000006801
 ### Set 14443A UID
 ^[Top](#top) ^^[Gen4](#g4top)
 
-UID is configured according to block0 with a backdoor write.
+UID is configured according to block0 with a backdoor write.  (Script commands are below the UID length examples)
 
-Example: preparing first two blocks:
+Example: preparing first two blocks: (Note the UMC has to be in MFC mode and the correct UID byte length set)
 ```
 hf 14a raw -s -c -t 1000 CF00000000CD00000102030405060708090A0B0C0D0E0F
 hf 14a raw -s -c -t 1000 CF00000000CD01101112131415161718191A1B1C1D1E1F
 hf 14a reader
 ```
-MFC mode, 4b UID  
+MFC 1k S50 4b UID  
 => UID `00010203`
+`script run hf_mf_ultimatecard -t 3 -u 00010203`
 
-MFC mode, 7b UID  
+MFC 4k S70 7b UID  
```

Comment:
hm...  mfc has s50 and s70 in 7b uid.   I don't recommend specifying it here.  
If your script sets the type to 4k,  then an extra line explaining would be better

Reply:
Removed.  It was just setting the ATQA/SAK to match.

Reply:
Thanks